src/Client.js

```diff
@@ -155,66 +165,76 @@ export default class Client extends EventTarget {
    * Get all the messages for the passed in alias
    *
    * Note the currently logged in account must own the alias associated with the aliasName.
-   * @param {string} aliasName the name of the alias to get messages for.
-   * @param {string[]} [interlocutors] an optional list of the senders and recipients of the messages.
-   * @param {Date} [sinceTime] an optional date to limit the request by. only receive messages since this date.
+   * @param {Object} options
+   * @param {string} options.aliasName the name of the alias to get messages for.
+   * @param {string[]} [options.interlocutors] an optional list of the senders and recipients of the messages.
+   * @param {Date} [options.sinceTime] an optional date to limit the request by. only receive messages since this date.
    * @returns {Promise<Message[]>} a list of messages which pass the filters.
    */
-  getMessagesForAlias(aliasName, interlocutors, sinceTime) {
+  getMessages({ aliasName, interlocutors, sinceTime } = {}) {
```

Comment:
`aliasName` is too weird, let's just call this `alias` or even username (I'm wondering if the API might make more sense if we frame it as primary username and secondary usernames and do away with alias as a concept entirely — just a thought)